packages/contracts/src/contracts/current/test/DummyERC721Token/DummyERC721Token.sol

```diff
@@ -0,0 +1,47 @@
+/*
+
+  Copyright 2018 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.4.21;
+import "/zeppelin/contracts/token/ERC721/ERC721Token.sol";
+import "../../utils/Ownable/Ownable.sol";
+
+contract DummyERC721Token is
+    Ownable,
+    ERC721Token
+{
+    function DummyERC721Token(
+        string name,
+        string symbol)
+        public
+        ERC721Token(name, symbol)
+    {}
```

Comment:
Do we want to keep the constructor if it's body is empty?

Reply:
@hysz are you saying that if a contract wants the constructor of it's base contract to be called, it must implement a constructor itself? If so, it is worth adding a comment above this explaining this quirk. That was non-obvious to me and could see a future developer remove this thinking it was superfluous.